HOWTO_meta_quest.md

```diff
@@ -352,46 +356,47 @@ properly capture the necessary content.
 ### 2. Change the Debug app to the GFXRecon Quest replay app
 
 If you haven't already done so, first follow the instructions to enable
-the debug layer in section
-[3. Enable GFXReconstruct](#3-enable-gfxreconstruct)
-in the Capturing IGL Content section above.
+the debug layer in the
+[Enable GFXReconstruct Vulkan Capture](#5-enable-gfxreconstruct-vulkan-capture)
+section above.
 
-One change is to make sure that we:
-* Capture the correct application (here it is the replay application)
+One change is to make sure that:
+* The correct application name is used (here it is the Quest replay
+  application)
 * Adjust the capture file to use a different name than
-the one being replayed (or you will overwrite what you're reading)
+  the one being replayed (or it will overwrite what is being read)
 
 ```bash
 adb shell settings put global gpu_debug_app com.lunarg.gfxreconstruct.replay
 adb shell "setprop debug.gfxrecon.capture_file  '/sdcard/Download/replay_capture.gfxr'"
 ```
 
-### 2. Install the Quest replay application
+### 2. Install the Updated Replay Tool
 
-(Follow instructions in section [1. Install the replay
-application](#1-install-the-replay-application)) in the Replay section
-above if you have not already done so.
+(Follow instructions in the 
+[1. Install the Quest Replay Application](#1-install-the-quest-replay-application))section above (especially if the Quest replay application was
+previously installed without re-capture enabled).
 
-As well as modifying the permissions as noted in section [2. Force
-External Storage Permissions](#2-force-external-storage-permissions)
-in the Replay section above.
+Repeat the permissions modification as noted in tje
+[2. Force External Storage Permissions](#2-force-external-storage-permissions)
+section above.
 
 ### 3. Run Logcat
 
 ```bash
-adb logcat -s IGL vulkan VulkanLoader VulkanLoaderAndroid OpenXR OpenXR-Loader gfxrecon
+adb logcat -s DEBUG vulkan VulkanLoader VulkanLoaderAndroid OpenXR OpenXR-Loader gfxrecon
```

Comment:
Include some {application name} or some other meta tag to show that application string filtering is useful.